src/index.js

```diff
@@ -88,4 +88,4 @@ icon.addEventListener('click', () => {
   window.location.reload();
 });
 
-displayTasks();
+displayTasks();
```

Comment:
:red_circle: [REQUIRED] Kindly ensure that when clearing completed items, corresponding removal from the localStorage is implemented. This aspect currently requires attention, as the existing functionality does not encompass this synchronization.

![no working](https://github.com/zhorabay/To-Do-list/assets/26736582/8a2db4bd-421b-464e-bf55-e6c37accabc2)